Emitron/Emitron/Constants.swift

```diff
@@ -31,7 +31,7 @@ import CoreGraphics.CGBase
 
 enum Constants {
   static let filters = "Filters"
-  static let clearAll = "Clear All"
+  static let resetFilters = "Reset filters"
```

Comment:
Nit - `filters` should be capitalised
```suggestion
  static let resetFilters = "Reset Filters"
```